wurzel/steps/qdrant/step.py

```diff
@@ -206,16 +239,94 @@ def _create_indices(self):
             field_schema=models.TextIndexParams(type=models.TextIndexType.TEXT, tokenizer=models.TokenizerType.WORD),
         )
 
-    def _retire_collections(self):
+    def _retire_collections(self) -> None:
+        """Retire (delete) historical Qdrant collections that.
+
+        - Are not among the last N (per COLLECTION_HISTORY_LEN),
+        - Are not currently targeted by an alias,
+        - Have not been recently used (per telemetry).
+
+        Skipped entirely if ENABLE_COLLECTION_RETIREMENT is False.
+        """
+        if self.settings.ENABLE_COLLECTION_RETIREMENT:
```

Comment:
Isn't this suppose to be "if not enabled"? The fact that no unit test caught this indicates that a unit test is missing.